ansible/plugins/connection/multi_passwd_ssh.py

```diff
@@ -55,6 +55,12 @@ def load_source(modname, filename):
           description: IPv6 address
           vars:
               - name: ansible_hostv6
+      host:
+          description: Hostname or IP address of the target machine.
+          vars:
+              - name: ansible_host
+      current_password_hash:
+          description: The hash of currently used password
```

Comment:
Is this change relevant?

Reply:
It's the new requirement of ansible, as we upgraded the ansible version in new image.